src/SFML/Window/OSX/SFKeyboardModifiersHelper.h

```diff
@@ -36,7 +36,7 @@ namespace priv
 {
 class WindowImplCocoa;
 }
-}
+} // namespace sf
```

Comment:
I'm guessing these got added because you used clang-format-15 to format. You can remove these since we now disallow using clang-format-15.

Reply:
Visual Studio formatted everything for me, and I guess it used clang-format-15.